src/hooks/__internal__/useCharacterCount/useCharacterCount.spec.tsx

```diff
@@ -1,24 +1,18 @@
 import * as React from "react";
```

Comment:
**issue**: since we have updated the spec for `useCharacerCount` like so:

```ts
const countValue = 10;
const countLimit = 20;
const [CharacterCount, visuallyHiddenHintId] = useCharacterCount(countValue, countLimit); 
```

these tests won't be functioning as intended, as on line 15, `maxLength` will actually be the `CharacterCount` component, and `characterCount` will actually be the id of the visually hidden hint text.

Reply:
Yeah this is a brilliant spot mate, obviously the tests were still passing but that's down to sheer luck on our end haha